docs/faq/index.md

```diff
@@ -1,3 +1,82 @@
+        "not-implemented": true
+            },
+```
+
+3\. unknown state (the ID got added but wasn't added to the file)
+```json
+    "some_concept": {
+        "name":"some concept name"
+        },
+```
+
+4\. concept exists (but no example code)
+```json
+    "some_concept": {
+    "name":"some concept name",
+    "comment": "optional comment",
+        },
+```
+
+## What if a developer could implement a workaround with code in the language to represent a concept? 
+Code Thesaurus is a language reference tool. It should explain how the language works and exists. This is not to invent functionality or to be an algorithm reference. If the concepts don't exist natively, they are not implemented in the language. For our purposes, they should all have "code" deleted and replaced with `"not-implemented": true`.
+
+## What should I put in the comments on a concept for the language? What are comments for?
+Comments provide additional detail on the implementation of a concept in a language. They are optional and important. For example, in the concept "String as an array of Characters" in Swift is implemented with String. At the time of this FAQ writing, the comment on this is "String in Swift bridges to Foundation's NSString class, so developers have a wide variety of string methods available to them." The comment provides a helpful commentary to point developers new to Swift to key information about the implementation.
+
+## Why do the concepts for maths not only show the operator, like **+**?
+For maths, the order of the operators may matter. Most languages usually use something along the lines of the operator in between two operands (i.e. a + b). Not all languages do, notably something like LISP does (+ a b).
+There are also prefix and postfix operators that might get confusing if people don't understand what prefix and postfix are. (As in what's the difference between ++ and ++ when we really mean ++a and a++?).
+
+Therefore, the concepts need to show that even though we use the same operator, like **+**, it is between or before or after the operands so that a developer reader could use the information to implement the concept in their own code.
+
+## What if I am assigned an Issue, but I can't finish it (as a contributor)?
+If you've started, do a PR for whatever you have accomplished. If you have not started, that's ok. In both instances, it helps to communicate. Make a comment on the Issue that you are no longer available, so someone else can get the assignment and the work can still be accomplished. More info at the [Code of Conduct](https://github.com/codethesaurus/codethesaur.us/blob/main/CODE_OF_CONDUCT.md). 
+
+## How long do I have to make my contribution PR?
+You should submit something within a week of being assigned. This lets the community know you are doing the work and keeps you assigned to the Issue. However, if you get unassigned and are still working, you can be reassigned. See [Contributing](contributing.md) for more info.
```

Comment:
I recently added a bot that will check in (send a warning) at 14 days/2 weeks, then 3 days later unassign if no one has said anything. All they have to do is leave a comment to keep the bot happy and reset the counter.